sigopt/requestor.py

```diff
@@ -39,6 +49,9 @@ def delete(self, url, params=None, json=None, headers=None):
 
   def request(self, method, url, params=None, json=None, headers=None):
     headers = self._with_default_headers(headers)
+    verify = self.verify_ssl_certs
+    if verify and self.ca_bundle:
+      verify = self.ca_bundle
```

Comment:
looks like `ca_bundle` is extraneous, can't you just use `verify` with the value of `ca_bundle`?

Reply:
Yep, just realized that's probably cleaner, and mimics what requests is doing anyway